docs/adr/0029-bulk-flow-segments.md

```diff
@@ -0,0 +1,67 @@
+---
+status: "draft"
+---
+# Ability to bulk create flow segments
+
+## Context and Problem Statement
+
+As part of the work on [ADR0024 source level edit](https://github.com/bbc/tams/blob/main/docs/adr/0024-source-level-edit.md) (also known as edit by reference) it was identified that it could be useful to be able to bulk create flow segments within the API. This ADR explores the impact of [option 5](https://github.com/bbc/tams/blob/main/docs/adr/0024-source-level-edit.md#option-5-provide-an-endpoint-to-bulk-write-flow-segments) this on the API implementation. 
```

Comment:
Typo?
```suggestion
As part of the work on [ADR0024 source level edit](https://github.com/bbc/tams/blob/main/docs/adr/0024-source-level-edit.md) (also known as edit by reference) it was identified that it could be useful to be able to bulk create flow segments within the API. This ADR explores the impact of [option 5](https://github.com/bbc/tams/blob/main/docs/adr/0024-source-level-edit.md#option-5-provide-an-endpoint-to-bulk-write-flow-segments) on the API implementation. 
```